templateflow/templateflow_get.py

```diff
@@ -0,0 +1,197 @@
+#!/opt/gensoft/exe/TemplateFlow_pyclient/0.8.0/venv/bin/python3.8
```

Comment:
This should have the standard apache header (which implicitly means you accept your contribution is covered by the license).

Reply:
my bad, I should have cleaned this sehbang.
I will